docs/getting_started/rendering_components.md

```diff
@@ -39,7 +39,7 @@ If you have embedded the component in a Django template using the
 
 You can simply render the template with the Django tooling:
 
-#### With [`django.shortcuts.render()`](https://docs.djangoproject.com/en/5.1/topics/http/shortcuts/#render)
+#### With [`django.shortcuts.render()`](https://docs.djangoproject.com/en/5.2/topics/http/shortcuts/#render)
```

Comment:
Btw so Django docs also have a URL format with `stable` instead of specific version number. So alternatively we could have set all links just to `stable`.

However, then if / once a link breaks, it would be harder to pinpoint when that occured, because there would not be any associated git commit.

This way, when all docs links are pinned, then if we next update the links version and run the link validation script, we will know exactly between which versions the error occured.

One example (tho not on this particular line of code) was that the Django setting `STORAGES` was called differently in Django 4.2. And so the URL fragment was correct on 4.2, but broken on 5.0, 5.1, 5.2.